types/user/user.ts

```diff
@@ -57,6 +58,10 @@ export interface User {
   courseEngagement: Score;
   projectDegree: Score;
   teamProjectDegree: Score;
+  hr: {
+    email: string;
+  };
+  status: Status;
```

Comment:
I can't see 'reservedTo'

Reply:
reservedTo has been added to User interface